### PR TITLE
added bottom nav bar and stated color

### PR DESCRIPTION
**Pull request by chris-saburo**
#240 
- added bottom nav bar in article page
- added enable status into each icon on bottom nav bar

### CODE REVIEW

**Comment by bamboochow** [comment]
ui approved @atharva3010 please review code

File: src/components/BaseNavigationBottom.vue [hunk 1]
```diff
@@ -140,25 +159,34 @@ ul.toggle-menu > li {
...
 
-.router-link {
+.router-normal-link {
   opacity: 0.54;
 }
 
-.router-link svg path {
-  fill: white;
+.router-normal-link svg path {
+  fill: #fff;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:177]: this too

File: src/components/BaseNavigationBottom.vue [hunk 2]
```diff
@@ -140,25 +159,34 @@ ul.toggle-menu > li {
...
   fill-opacity: 0.54;
 }
 
+.router-active-link {
+  opacity: 1;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:182]: no need of specifying a class for this

**Comment by atharva3010** [comment]
I can recall having this discussion once before with @mort3za , on one of your PRs (https://github.com/whynotearth/Volkswagen-email-corporate-communications-CMS/pull/246). 
That the solution used here is not a good way of showing active status in links, neither repeating `text-white` makes any sense. It's discouraging as a reviewer to find the same repeated issues again and again.

File: src/components/BaseNavigationBottom.vue [hunk 3]
```diff
@@ -6,25 +6,40 @@
         <div class="flex relative">
           <ul class="flex-auto flex justify-around text-left menu list-none bg-primary px-2 md:px-0">
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'Dashboard' }">
+              <router-link
+                :class="active == 'dashboard' ? 'text-white router-active-link' : 'text-white router-normal-link'"
```
[the remark this answers is not in the document]
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:10]: Done it

File: src/components/BaseNavigationBottom.vue [hunk 4]
```diff
@@ -6,25 +6,40 @@
...
+                :to="{ name: 'Dashboard' }"
+              >
                 <HomeIcon class="m-auto" />
                 Home
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'Stats' }">
+              <router-link
+                :class="active == 'stats' ? 'text-white router-active-link' : 'text-white router-normal-link'"
```
[the remark this answers is not in the document]
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:19]: Done it.

File: src/components/BaseNavigationBottom.vue [hunk 5]
```diff
@@ -6,25 +6,40 @@
...
+                :to="{ name: 'Stats' }"
+              >
                 <StatsIcon class="m-auto" />
                 Stats
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'JumpStartLists' }">
+              <router-link
+                :class="active == 'bluedelta' ? 'text-white router-active-link' : 'text-white router-normal-link'"
```
[the remark this answers is not in the document]
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:29]: Done it.

File: src/components/BaseNavigationBottom.vue [hunk 6]
```diff
@@ -6,25 +6,40 @@
...
+                :to="{ name: 'JumpStartLists' }"
+              >
                 <MemoStatslIcon class="m-auto" />
                 Blue Delta
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'ArticleLists' }">
+              <router-link
+                :class="active == 'article' ? 'text-white router-active-link' : 'text-white router-normal-link'"
```
[the remark this answers is not in the document]
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:39]: Done it.

File: src/components/BaseNavigationBottom.vue (hunk 2 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue]: that style should exist.

**Comment by mort3za** [comment]
I see the same issues that Atharva mentioned.
Style links normally, then override styles in active mode

**Pull request merged**